site/content/gitops-quickstart/01-apply-gitops.md

```diff
@@ -187,9 +187,29 @@ in your cluster:
 - [Fluentd](https://www.fluentd.org/) & Amazon's [CloudWatch agent](https://aws.amazon.com/cloudwatch/) -- for cluster & containers' [log collection, aggregation & analytics in CloudWatch](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-setup-logs.html).
 - [podinfo](https://github.com/stefanprodan/podinfo) --  a toy demo application.
 
-- xxx: How users should check what is running at the end of the process
-  (e.g. get pods, port forward to kube dashboard, grafana, how they get
-  to the demo app that has been deployed)
+It's easy to confirm if all of this is up and running for you. Let's
+check `podinfo` and see if it's up.
+
+```console
+kubectl get svc -n demo
```

Comment:
For guides and scripts I typically prefer the long version of CLI arguments for explicitness:

```suggestion
kubectl get service --namespace demo
```

Or maybe everyone knows what the `-n` is?